### PR TITLE
[ECO-5786] Make use of async await wherever makes sense

### PR DESCRIPTION
**Pull request by emilianop11**

### CODE REVIEW

File: server/serverMethods.js [hunk 1]
```diff
@@ -639,23 +636,20 @@ function ServerMethods(aLogLevel, aModules) {
...
 
-  function lockRoom(aReq, aRes) {
+  async function lockRoom(aReq, aRes) {
     var roomName = aReq.params.roomName.toLowerCase();
-    serverPersistence
-      // eslint-disable-next-line consistent-return
-      .getKey(redisRoomPrefix + roomName).then((room) => {
-        if (!room) return aRes.status(404).send(null);
+    let room = await serverPersistence.getKey(redisRoomPrefix + roomName);
+    if (!room) return aRes.status(404).send(null);
```
Comment (Raulkg) [inline server/serverMethods.js:643]: i think we can improve this by sending error response instead of null

**Review by Raulkg — COMMENTED** [review]
Are we only making this change in server-side methods?

File: server/serverMethods.js (hunk 1 above)
Comment (emilianop11) [reply server/serverMethods.js:643]: I agree, but lets do it in a separate PR to keep concerns separate

**Comment by emilianop11** [comment]
Per our discussion during the daily standup, we'll be doing this only server side

**Review by Raulkg — APPROVED** [review]
LGTM

File: server/serverMethods.js (hunk 1 above)
Comment (emilianop11) [reply server/serverMethods.js:643]: https://tokbox.atlassian.net/browse/ECO-5793

**Pull request merged**